README.md

```diff
@@ -1,181 +1,54 @@
-# Tracee
-Container and system tracing using eBPF
+# Tracee - System tracing using eBPF
 
-**Tracee** is a lightweight, easy to use container and system tracing tool.
-After launching the tool, it will start collecting traces of newly created containers (container mode) or processes (system mode).
-The collected traces are mostly system calls performed by the processes,
-but other events, such as capabilities required to perform the actions requested by the process, are also supported.
+Tracee is a lightweight and easy to use system tracing tool. It allows you to observe system calls and other system events in real time.
 
-Tracee CLI was originally written in Python, but is now being ported to Go. Currently both versions are available in the repo. Python is considered more stable and tested, but future development will be in Go. This readme is referring to the Python version as it's recommended for newcomers but all of the features are similarly supported in the Go version, give or take minor CLI differences.
+Tracee CLI was originally written in Python, but was since ported to Go. Currently both versions are still available in the repo, but future development will be in Go and the Python version will eventually be deprecated and removed.
 
-## Requirements
-Currently requires 
-* kernel version > 4.14
-* BCC
-* Python if using the Python version
+## Getting started
 
-## Quick Start Instructions
+### Prerequisites
+To run, Tracee requires the following:
+* Linux kernel version > 4.14
+* Kernel headers
+* C standard library (currently tested with glibc)
+* [BCC](https://github.com/iovisor/bcc)
 
-As root: `start.py [-h] [-c] [-b PAGE_NR] [-j] [-l] [-e EVENTS_TO_TRACE]`
+### Released artifacts
+Currently we don't yet have a release process for Tracee. You can build Tracee from source using `make build` or use the Docker image: `aquasec/tracee:latest` from Docker Hub.
 
-optional arguments:
-
--h, --help            show this help message and exit
-
--c, --container       only trace newly created containers
-
--b {1,2,4,8,16,32,64,128,256,512,1024}, --buf-pages {1,2,4,8,16,32,64,128,256,512,1024}
-                      number of pages for perf buffer, defaults to 64
-
--j, --json            save events in json format
-
--l, --list            list events
-
--e EVENTS_TO_TRACE, --events-to-trace EVENTS_TO_TRACE
-trace only the specified events and syscalls (default: trace all)
-
---show-syscall        show syscall name in kprobes
-
---exec-env            show execve(at) environment variables in output
-
-
-examples:
-
-`./start.py -c`
-
-Following is an output example of Tracee after running
-
-`docker run -it --rm alpine sh`
+### Quickstart
+Following is a quick start tutorial on Ubuntu VM with Docker.
+Ubuntu is a convenient image which includes all of the host requirements (kernel, headers, glibc). BCC is included in the tracee container image.
```

Comment:
that's not what I meant, perhaps the word "image" is misused here. I meant to explain why the quickstart is using Ubuntu, the operating system, because it satisfies our prerequisites. I didn't mean to discuss the tracee docker image.

Reply:
actually the base image for tracee does not include the kernel headers - those should be mounted from the host.
@lizrice  Would it make sense if I just change the opening:

> Ubuntu is used here as it conveniently includes all of the host requirements (kernel, headers, glibc). The BCC requirement is bundled into the tracee container image.

Reply:
https://github.com/aquasecurity/tracee/pull/70/commits/2e83eeb8398ed67ed3cd4403ab5b58d6acc52b87